plugins/modules/mssql_script.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2014, Vedit Firat Arig <firatarig@gmail.com>
+# Outline and parts are reused from Mark Theunissen's mysql_db module
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: mssql_script
+short_description: Run simple, file based, or templated mssql scripts
+description:
+   - Run simple, file based, or templated mssql scripts
```

Comment:
```suggestion
   - Run simple, file based, or templated mssql scripts.
```
it doesn't have any file related options, should we reformulate the description?

Reply:
I can add into examples, just need to use lookup plugin. I was failing a yaml sanity check before with that example so I removed.